src/zentables/zentables.py

```diff
@@ -505,10 +505,10 @@ def _internal_freq_table(
         **kwargs,
     ) -> pd.DataFrame:
 
-        if not totals or not subtotals:
-            warnings.warn(
-                "`props` is set. Overriding `totals` and `subtotals` settings."
-            )
+        # if not totals or not subtotals:
```

Comment:
Why are you commenting this warning out?

Reply:
It was flooding my terminal, so I temporarily commented it out.
Will uncomment it now!